src/cmd/manage_nsfs.js

```diff
@@ -544,16 +570,17 @@ async function get_account_config_file_status(data, accounts_path, access_keys_p
     }
 }
 
-
 async function manage_account_operations(action, data, config_root, config_root_backend) {
     const accounts_path = path.join(config_root, accounts_dir_name);
     const access_keys_path = path.join(config_root, access_keys_dir_name);
     if (action === 'add') {
         await add_account_config_file(data, accounts_path, access_keys_path, config_root_backend);
```

Comment:
I think you need to remove the validation for access_key and secret_key.

Reply:
The validation should be still there if those fields do not exist (bug) then it should fail.
I do agree that it should never fail.